services/dask-sidecar/src/simcore_service_dask_sidecar/tasks.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from uuid import UUID
+
+from simcore_service_sidecar.cli import run_sidecar
+from simcore_service_sidecar.utils import wrap_async_call
+
+from .meta import print_banner
+from .settings import Settings
+
+log = logging.getLogger(__name__)
+
+
+print_banner()
+
+
+def test(x):
+    return x + 1
+
```

Comment:
what is this for?

Reply:
it adds one to x ;-b
No, seriously, this is just as trivial check. I am scheduling dask tasks from a jupyter notebook (special dynamic service I am building separately) and this is a trivial task that I use.
Will probably disappear in next version

Reply:
maybe just add a comment to explain the usage